build.assets/Makefile

```diff
@@ -189,10 +222,21 @@ buildbox-centos7:
 # Builds a Docker buildbox for CentOS 7 FIPS builds
 # Uses regular CentOS 7 buildbox as base, so limited arguments are needed.
 #
+# Because this target uses the image built by the buildbox-centos7 target, it is
+# expected to be run using a docker buildx driver. If another driver is used, the
```

Comment:
s/a docker/the "docker"/

That currently reads like it just needs some docker buildx driver rather than the specific "docker" buildx driver.

Reply:
9c41a6810d1eedbf1a9f170ba4c7e8ed6075c288

Reply:
This is not address in that commit i think?

Reply:
As of that commit it says

> # DOCKER_BUILDER_NAME extracts the name of the buildx builder that uses the "docker" driver.

I think that addresses the original concern, but maybe I didn't understand it properly? Feel free to suggest something else.

Reply:
That's a different comment - this one says to replace 
> a docker buildx driver

with

> the "docker" buildx driver

Reply:
that is, in this sentence where my comment is attached, not in the heading.

Reply:
Oh my bad: 75b79ec889e8b02d468ef3bb6d17d37571ed2f9c